static/js/services/image-metadata-service.js

```diff
@@ -27,5 +27,14 @@ angular.module('quay').factory('ImageMetadataService', [function() {
     return found.command;
   };
 
+  metadataService.getManifestCommand = function(manifest, blobDigest) {
```

Comment:
Clair v4 returns `added_by` for each feature using the layer's `blob_digest`, not the `imageID` as in Clair v2.

Reply:
Yep